policyengine_us/variables/gov/states/nm/tax/income/deductions/nm_deductions.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class nm_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New Mexico income deductions"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.NM
+
+    adds = [
```

Comment:
Can't take both standard and itemized deductions; this will need a formula

Reply:
I tried to use "tax_unit_itemizes" as a condition but it seems to create a circular reference, is there a way to fix that?